components/alert.css

```diff
@@ -0,0 +1,101 @@
+.alert {
+  display: flex;
+  flex-basis: 90%;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 0.5rem;
+  padding: 1rem;
+  border-radius: 0.2rem;
+  color: var(--alert-primary);
+  background-color: var(--alert-primary-bg);
+}
+
+/* || Default Alert || */
+.primary {
+  color: var(--alert-primary);
+  background-color: var(--alert-primary-bg);
+}
+
+.secondary {
+  color: var(--alert-secondary);
+  background-color: var(--alert-secondary-bg);
+}
+
+.success {
+  color: var(--alert-success);
+  background-color: var(--alert-success-bg);
+}
+
+.warning {
+  color: var(--alert-warning);
+  background-color: var(--alert-warning-bg);
+}
+
+.error {
+  color: var(--alert-error);
+  background-color: var(--alert-error-bg);
+}
+
+.info {
+  color: var(--alert-info);
+  background-color: var(--alert-info-bg);
+}
+
+/* || Outline Alert || */
+.primary-outline {
+  color: var(--alert-primary);
+  border: 0.01rem solid var(--alert-primary);
+  background-color: var(--white);
+}
+
+.secondary-outline {
+  color: var(--black);
+  border: 0.01rem solid var(--black);
+  background-color: var(--white);
+}
+
+.success-outline {
+  color: var(--alert-success);
+  border: 0.01rem solid var(--alert-success);
+  background-color: var(--white);
+}
+
+.warning-outline {
+  color: var(--alert-warning);
+  border: 0.01rem solid var(--alert-warning);
+  background-color: var(--white);
+}
+
+.error-outline {
+  color: var(--alert-error);
+  border: 0.01rem solid var(--alert-error);
+  background-color: var(--white);
+}
+
+.info-outline {
+  color: var(--alert-info);
+  border: 0.01rem solid var(--alert-info);
+  background-color: #fff;
```

Comment:
I think you forgot to use var to define color here.

Reply:
Thank you for pointing that out. Will correct